src/tm_mad/ceph/postmigrate

```diff
@@ -55,6 +70,8 @@ fi
 
 migrate_other "$@"
 
-exec_and_log "$SSH $SRC_HOST rm -rf $DST_PATH"
+if [ "$SHARED" != "YES" ]; then
```

Comment:
OK. I can change it.
But won't it break the code style and create no more confusion there?